package.json

```diff
@@ -4,47 +4,27 @@
-    "functional:local": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
-    "functional:cron:prod": "ALLOY_ENV=\"prod\" testcafe chrome",
-    "functional:local:allure": "testcafe chrome -r allure",
-    "functional:local:headless": "testcafe chrome:headless",
-    "functional:local:build": "npm run build:prod && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
-    "functional:local:watch": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" BUILD=\"prodStandalone\" ./scripts/watchFunctionalTests.js",
-    "functional:api": "testcafe chrome:headless",
-    "prebuild": "npm run clean && npm run format && npm run lint",
-    "build": "rollup -c",
-    "prebuild:prod": "npm run prebuild",
-    "build:prod:standalone:unminified": "rollup -c --environment BUILD:prodStandalone",
-    "build:prod:standalone:minified": "rollup -c --environment MINIFY,BUILD:prodStandalone",
-    "build:prod:reactor:unminified": "rollup -c --environment BUILD:prodReactor",
-    "build:prod": "npm-run-all --parallel build:prod:standalone:unminified build:prod:standalone:minified build:prod:reactor:unminified",
-    "build:watch": "rollup -c -w",
-    "build:basecode": "terser src/baseCode/index.js --mangle --compress unused=false",
-    "sandbox": "cd sandbox && npm start",
-    "sandbox:install": "cd sandbox && npm install",
-    "sandbox:build": "cd sandbox && npm run build",
-    "dev": "npm-run-all --parallel build:watch sandbox",
-    "checkbundlesize": "bundlesize",
-    "mockserver": "./scripts/mockServer.js"
+    "test": "npm run test:unit && npm run test:functional",
+    "test:unit": "karma start --single-run",
+    "test:unit:watch": "karma start",
+    "test:unit:saucelabs": "karma start karma-saucelabs.conf.js",
+    "test:unit:coverage": "karma start --single-run --reporters spec,coverage",
+    "test:functional": "rollup -c --environment BUILD:prodStandalone && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
+    "test:functional:watch": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" BUILD=\"prodStandalone\" ./scripts/watchFunctionalTests.js",
+    "test:functional:ci": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:ie@11:Windows 10'",
+    "test:functional:cron:prod": "ALLOY_ENV=\"prod\" testcafe chrome",
+    "test:functional:api": "testcafe chrome:headless",
+    "dev": "concurrently --names build,sandbox \"rollup -c -w\" \"cd sandbox && npm start\"",
+    "build": "npm run format && npm run lint && npm run clean && concurrently \"rollup -c --environment BUILD:prodStandalone\" \"rollup -c --environment MINIFY,BUILD:prodStandalone\" \"rollup -c --environment BUILD:prodReactor\" && echo \"Base Code:\" && terser src/baseCode/index.js --mangle --compress unused=false",
+    "checkthattestfilesexist": "./scripts/checkThatTestFilesExist.js"
```

Comment:
I decided to keep this as an npm script instead of only running it on the pre-commit hook. When I create a bunch of new modules, it's helpful to understand which spec files I still need to create (without having to try to commit first).